components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/constant/SQLConstants.java

```diff
@@ -165,6 +238,66 @@
             "  R.NAME = ?\n" +
             "  AND R.TENANT_ID = ?\n" +
             "  AND R.TYPE_ID = ?";
+    public static final String GET_RESOURCE_BY_NAME_MSSQL = "SELECT " +
+            "  R.ID," +
+            "  R.TENANT_ID," +
+            "  R.NAME," +
+            "  R.CREATED_TIME," +
+            "  R.LAST_MODIFIED," +
+            "  R.HAS_FILE," +
+            "  R.HAS_ATTRIBUTE," +
+            "  T.NAME AS RESOURCE_TYPE," +
+            "  T.DESCRIPTION AS DESCRIPTION," +
+            "  F.ID AS FILE_ID," +
+            "  F.NAME AS FILE_NAME," +
+            "  A.ID AS ATTR_ID," +
+            "  A.ATTR_KEY AS ATTR_KEY," +
+            "  A.ATTR_VALUE AS ATTR_VALUE" +
+            "FROM" +
+            "  IDN_CONFIG_RESOURCE AS R" +
+            "  INNER JOIN IDN_CONFIG_TYPE AS T ON R.TYPE_ID = T.ID" +
```

Comment:
Dropping the keyword `AS` should also work with MySQL AFAIR. If so, we might be able to reuse the oracle query.

Reply:
Please check similar usages.

Reply:
Used the same query for ORACLE and MSSQL. Changed in 5a0504e
Due to the following difference there is a separate query for mysql
R.HAS_ATTRIBUTE = 1 , R.HAS_FILE = 1 -> in mssql and oracle
R.HAS_ATTRIBUTE = TRUE, R.HAS_FILE = TRUE -> mysql